jiant/config/defaults.conf

```diff
@@ -161,7 +161,12 @@ val_data_limit = 5000  // Maximum number of examples to be used during mid-train
                        // by do_full_eval.
 val_interval = 1000  // Number of gradient steps to take between validation checks. Note that
                      // the stopping criteria are only checked at these intervals.
-
+accumulation_steps = 1 // The number of batches for which to accumulate gradients before taking
+                       // an optimizer step. If accumulation_steps = 2, then two batches/updates
+                       // are processed before taking one optimizer step. Training metrics (e.g.,
+                       // loss) are reported at step intervals, i.e., training with batch_size = n
+                       // and accumulation_steps = 2 approximates the training process with
```

Comment:
approximates => simulates ... with a lower peak memory usage and a longer step time